UVAtlasTool/Mesh.cpp

```diff
@@ -1087,6 +1089,59 @@ HRESULT Mesh::GetVertexBuffer(_Inout_ DirectX::VBWriter& writer) const
     return S_OK;
 }
 
+//--------------------------------------------------------------------------------------
+void Mesh::ExportToOBJ(const wchar_t* szFileName) const
+{
+    std::wstring ws(szFileName);
+    ExportToOBJ(std::string(ws.begin(), ws.end()));
+}
+/// Write to file
```

Comment:
Get rid of the narrow-string version of this method. With Visual C++ you can use a ``wchar_t*`` directly with ``ofstream::open``

Reply:
done. Made the main method receive `const wchar_t* szFileName`